pkg/securitypolicy/framework.rego

```diff
@@ -1778,6 +1817,17 @@ allow_unencrypted_scratch := data.policy.allow_unencrypted_scratch
 default allow_capability_dropping := false
 
 allow_capability_dropping := flag {
-    semver.compare(data.policy.framework_svn, "0.2.2") >= 0
+    semver.compare(policy_framework_version, "0.2.2") >= 0
     flag := data.policy.allow_capability_dropping
 }
+
+default policy_framework_version := null
+default policy_api_version := null
+
+policy_framework_version := data.policy.framework_version
+policy_api_version := data.policy.api_version
+
+# deprecated
+policy_framework_version := data.policy.framework_svn
```

Comment:
Throwing an error is the correct behaviour. If a policy contains both of these, then it is by definition invalid. In this case, this will produce the nice behaviour that the policy won't even compile.